wptool/wptool.cs

```diff
@@ -180,6 +180,7 @@ class wptool
 					if (command == "launch")
 					{
 						IRemoteApplication app = device.GetApplication(appid);
+						app.TerminateRunningInstances();
```

Comment:
Note: I was thinking if we could check if it's running like below:

```c++
if (app.IsRunning()) {
    app.TerminateRunningInstances();
}
```

But this didn't work, unfortunately `IsRunning()` always throws Exception with `The method or operation is not implemented.` for me.